pipelines/push-to-external-registry/push-to-external-registry.yaml

```diff
@@ -221,6 +221,23 @@ spec:
           workspace: release-workspace
       runAfter:
         - verify-enterprise-contract
+    - name: update-cr-status
```

Comment:
I think we want this in the finally. If we pass like push-snapshot but fail later on, we should still update the release status. If this fails and we skip cleanup-workspace because of it, that is bad, but :man_shrugging:

Reply:
Nice that you said that. I was with exactly this doubt.

Reply:
https://tekton.dev/docs/pipelines/pipelines/#cannot-configure-the-finally-task-execution-order or maybe we can't, as we can't have cleanup-workspace run before this since this reads from the workspace

Reply:
Resolving this as per the restrictions, having it be the last thing to run (before the finally) is the best we can do

Reply:
ah ok. I was moving it to finally - I saw that but thought you was taking it as a risk under control. I remove the last commit.